src/dial.js

```diff
@@ -168,11 +168,9 @@ module.exports = function dial (swarm) {
           if (err) {
             return callback(err)
           }
-
-          pt.setReadable(conn)
-          pt.setWritable(conn)
-          pt.peerId = pi.id
-          callback(null, pt)
+          proxyConn.setInnerConn(conn)
+          proxyConn.peerId = pi.id
```

Comment:
This is 'pre Identify factor out', after the refactor it should always be `getPeerInfo`